layers/lispy/packages.el

```diff
@@ -0,0 +1,74 @@
+;;; packages.el --- lispy layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author:  <burketje@BURKETJE-2014>
```

Comment:
Is this intended? Should probably be your name <your email>. 
It was decided to pay more attention Author: field while reviewing new layers!